client/app/dbaas/logs/detail/options/logs-options.service.js

```diff
@@ -1,26 +1,25 @@
 class LogsOptionsService {
```

Comment:
Most of the methods in this file have been alphabetically re-ordered. Apart from that, getSubscribedOptionsMapGrouped is a newly added method that groups options by type. Additionally, the options are being sorted alphabetically now.